Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -646,139 +646,139 @@ Don't have a 4chan Pass?<br>
+
+    <string name="site_uses_dynamic_boards">"Since the site likely has dynamic boards you must add one from the \"Boards\" menu first."</string>
+    <string name="could_not_start_saving_no_permissions">Grant write access permission to start downloading images</string>
+    <string name="update_manager_copy_apk_title">Copy APK?</string>
+    <string name="update_manager_copy_apk">Would you like to store this APK?</string>
+    <string name="update_manager_could_not_convert_uri">Could not convert URI (%1$s) into an ExternalFile</string>
+    <string name="update_manager_input_file_does_not_exist">Input APK file (%1$s) does not exist</string>
     <string name="update_manager_output_file_does_not_exist">Output file (%1$s) does not exist</string>
-    <string name="update_manager_could_not_copy_apk">Couldn\'t copy file contents from file (%1$s) into (%2$s)</string>
-    <string name="update_manager_apk_copied">Apk successfully copied</string>
-    <string name="settings_show_copy_apk_dialog_title">Show copy apk dialog when downloading an update</string>
-    <string name="settings_show_copy_apk_dialog">Every time you download a new update a dialog with suggestion to copy that apk to some other directory will be shown</string>
+    <string name="update_manager_could_not_copy_apk">Could not copy file contents from file (%1$s) into (%2$s)</string>
+    <string name="update_manager_apk_copied">APK copied</string>
+    <string name="settings_show_copy_apk_dialog_title">Show \"copy APK\" dialog when downloading an update</string>
+    <string name="settings_show_copy_apk_dialog">Every time you download a new update a dialog with suggestion to copy that APK to some other directory will be shown</string>
 
-    <string name="could_not_initialized_captcha">Couldn\'t initialize captcha, reason: %1$s</string>
+    <string name="could_not_initialized_captcha">Could not present CAPTCHA because %1$s</string>
     <string name="fail_reason_webview_is_not_installed">WebView is not installed</string>
-    <string name="fail_reason_some_part_of_webview_not_initialized">Some part of WebViewChromium couldn\'t get initialized: (%1$s)</string>
+    <string name="fail_reason_some_part_of_webview_not_initialized">Could not start some part of WebViewChromium because %1$s</string>
 
-    <string name="setting_enable_image_link_loading">Enable image link loading</string>
-    <string name="setting_enable_image_link_loading_description">Adds any JPG, PNG, GIF, WEBM, MP4, PDF, BMP, WEBP, MP3, SWF, M4A, OGG, or FLAC files linked in posts as internally viewable post attachments with a special spoiler image.</string>
+    <string name="setting_enable_image_link_loading">Image link loading</string>
+    <string name="setting_enable_image_link_loading_description">Adds any JPG, PNG, GIF, WebM, MP4, PDF, BMP, WebP, MP3, SWF, M4A, OGG, or FLAC files linked in posts as internally viewable post attachments with a special spoiler image.</string>
```

Comment:
As per https://en.wikipedia.org/wiki/WebM and https://en.wikipedia.org/wiki/WebP